static/app/views/issueDetails/header.tsx

```diff
@@ -162,6 +162,11 @@ function GroupHeaderTabs({
   );
 }
 
+function sanitizeIssueBreadcrumbQuery(query: string): string {
+  const sanitizedQuery = query.replace('&sort=timestamp', '');
```

Comment:
Manipulating the string is error-prone, but luckily you can access the query string as an object like so:

```
const location = useLocation();

const query = location.query
```

And when you link to it, you can pass an object instead of a string:

```
to: { pathname: '/organizations/whatever/...', query: {...} }
```

Reply:
May I jump in to say that the original issue point at a specific query param, although it would also break for any query params that is added from the previous view such as `Sort key '-runtime' not supported.` and many more.
You may want to filter out the ones that a certain view does not support may be (intersection of the filters)? however from the particular use case of moving from `/issues/123/events/` to `/issues` I don't think there is any filters that would work on both.

Cheers,

Reply:
@malwilley @AdrienFromToulouse Thank you both for your feedback!